Update main references in histogram tutorial (#598)
mozilla/data-docs: src/cookbooks/main_ping_exponential_histograms.md

This tutorial tries to build up an understanding and intuition of how things work on a low-level before it gets to its main act of reproducing GLAM. If you don't care about the details, you can probably skip the earlier sections in this document.

Assumptions: 
    - You have some idea of what a histogram is (if not, the [Wikipedia article](https://en.wikipedia.org/wiki/Histogram) is a great place to start), have at least skimmed over [the Firefox documentation on histograms](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/collection/histograms.html)
    - You have read [the introduction to STMO](../tools/stmo.md).

>     - You understand the [Main Ping tables](../datasets/main_ping_tables.md)

Reviewer: ```suggestion
    - You understand the [main ping tables](../datasets/main_ping_tables.md) (`telemetry.main_1pct` and `telemetry.main_nightly`).
```

SELECT
  payload.histograms.FX_TAB_SWITCH_SPINNER_VISIBLE_MS AS histogram_json,
FROM

>   telemetry.main_nightly -- Use telemetry.main_1pct for a 1% sample of non-nightly data

Reviewer 2: ```suggestion
  telemetry.main_nightly -- Use telemetry.main_1pct for a 1% sample across channels
```

Reviewer 2: And we should probably change the `sample_id` value to `0` to be compatible with `main_1pct`. If a user tries to query `main_1pct` with the existing filters here, they will get an empty result set.

Reviewer: I think we should at least introduce the concept of `main_nightly` if we're going to use it in the example. A quick sentence above (with a link to more information, if available?) should be fine.

WHERE
  sample_id = 42

>   AND normalized_channel = 'nightly'

Reviewer: ```suggestion
  AND normalized_channel = 'nightly' -- Only technically necessary if using telemetry.main or telemetry.main_1pct (see above)
```

Can you also change the queries below to incorporate this?
